src/components/PeopleTable/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person,
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const { name, slug, sex } = person;
+
+  return (
+    <Link to={`../${slug}`} className={classNames({ 'has-text-danger': sex === 'f' })}>
```

Comment:
Because `className` is long better rewrite it this way 👇  Would be easier to read.
```suggestion
    <Link 
      to={`../${slug}`} 
      className={classNames({ 'has-text-danger': sex === 'f' })}
    >
```